### DISCUSSION TITLE
posts/drain-a-well/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#41] [Comments]
# posts/drain-a-well/

清明回家，家里装修还剩下一片水泥地没有浇。为了浇水泥地，早早地把原本井边的水泥凿掉，安装了手动抽水泵，这也导致井边的水会不断渗回井里，导致井水有异味。解决方案是把现有水井抽干，让新的地下水渗到井里；如果想更彻底一些，可以多重复几次，亦或是放点漂白粉或者石灰，搅拌均匀后再抽干。
[&hellip;] 老爸说等水泥地浇好再来抽干，但我又是闲不住啦！搭建了简易的排水渠，吭哧吭哧压起抽水泵来。家里的井是三 &hellip;

https://shitao5.org/posts/drain-a-well/

**Comment by yihui** [comment]
刚试了一下你的数据分析工具箱上的二维码，结果弹出来提示框说“根据风险管理控制要求，暂时不支持向该个人二维码付款”，不知道是我微信的问题还是你的。前几天我刚试过给别人捐款，因为额度太大被拒了，现在我也不知道是不是我的微信支付功能被限制了。

**Reply by Shitao5** [reply]
谢谢 yihui！刚刚拿小王的微信试了下是可以支付成功的。查了下[微信支付用户服务协议](https://weixin.qq.com/cgi-bin/readtemplate?t=weixin_business_pay)，第 2.1.4 里边写的:

> A、你办理充值、收取或支付单笔金额人民币1万元以上或者外币等值1000美元以上业务的；
>
> B、你全部微信支付账户项下30天内充值、收取及支付总金额累计人民币5 万元以上或外币等值1万美元以上的；
> 
> C、你全部微信支付账户余额连续10天超过人民币5000元或外币等值1000美元的；
> D、你使用本服务买卖金融产品或服务的；
> 
> E、本公司认为你的身份信息或相关资料存在疑点，或交易行为、交易情况出现异常的；
> 
> F、本公司认为应核对或留存你身份证件或其他必要文件的其他情形的。

我也是第一次遇到这种情况，细看下协议卡挺严的，看来俺也要去搞个爱发电啥的。

**Reply by yihui** [reply]
刚又试了一下，一块钱可以，一百就不行，连十块都不行……但我又试了向其他人支付或转账，都正常。可能是我前几天向陌生人捐了超大额度引起了微信的怀疑，现在不让我向陌生二维码支付了。

**Reply by Shitao5** [reply]
收到一块，谢谢 yihui！第一次收到捐赠很开心哈哈。

超大额度肯定引起微信注意了，微信：I'm watching! 👀。安全和便利的权衡之间，微信平台选择了前者，这无可厚非。益辉现在像是一个基金会哈哈哈。

**Reply by Shitao5** [reply]
等机缘巧合到了，欢迎yihui扫微信支付再上面那个码😉
